projects/workshop/rick_and_morty_app/README.md

```
@@ -0,0 +1,86 @@
+    stackTrace: stackTrace,
+);
+```
+
+## Pacotes utilizados:
+
+- **go_router**: Utilizado para a navegação de páginas.
+- **dio**: Para requisições http.
+- **provider**: Para injeções de dependências.
+- **intl**: Para manipulação de timestamp.
+- **google_fonts**: Para utilizar a fonte ``Lato`` seguindo os requisitos de design.
+- **mocktail**: Para testes.
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)